controllers/categories.controller.js

```diff
@@ -1,8 +1,24 @@
-const {modelGetCategories} = require("../models/categories.model.js");
-
+const {
+  modelGetCategories,
+  modelGetEndpoints,
+} = require("../models/categories.model.js");
 
 exports.getCategories = (req, res) => {
-    modelGetCategories().then((categories) => {
-        res.status(200).send({categories});
+  modelGetCategories()
+    .then((categories) => {
+      res.status(200).send({ categories });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.getEndpoints = (req, res) => {
+  modelGetEndpoints()
```

Comment:
I would suggest having a model for this is unnecessary. Happy for you to leave as is though.